osgp/platform/osgp-adapter-domain-core/src/test/java/org/opensmartgridplatform/adapter/domain/core/application/services/DeviceInstallationServiceTest.java

```diff
@@ -0,0 +1,276 @@
+import org.opensmartgridplatform.domain.core.entities.Device;
+import org.opensmartgridplatform.domain.core.entities.LightMeasurementDevice;
+import org.opensmartgridplatform.domain.core.entities.Ssld;
+import org.opensmartgridplatform.domain.core.exceptions.ValidationException;
+import org.opensmartgridplatform.domain.core.repositories.SsldRepository;
+import org.opensmartgridplatform.domain.core.services.DeviceDomainService;
+import org.opensmartgridplatform.domain.core.services.OrganisationDomainService;
+import org.opensmartgridplatform.domain.core.valueobjects.DeviceStatus;
+import org.opensmartgridplatform.domain.core.valueobjects.DeviceStatusMapped;
+import org.opensmartgridplatform.domain.core.valueobjects.LightType;
+import org.opensmartgridplatform.domain.core.valueobjects.LightValue;
+import org.opensmartgridplatform.domain.core.valueobjects.LinkType;
+import org.opensmartgridplatform.domain.core.valueobjects.TariffValue;
+import org.opensmartgridplatform.shared.exceptionhandling.ComponentType;
+import org.opensmartgridplatform.shared.exceptionhandling.FunctionalException;
+import org.opensmartgridplatform.shared.exceptionhandling.NoDeviceResponseException;
+import org.opensmartgridplatform.shared.exceptionhandling.TechnicalException;
+import org.opensmartgridplatform.shared.infra.jms.CorrelationIds;
+import org.opensmartgridplatform.shared.infra.jms.RequestMessage;
+import org.opensmartgridplatform.shared.infra.jms.ResponseMessage;
+import org.opensmartgridplatform.shared.infra.jms.ResponseMessageResultType;
+
+@ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.LENIENT)
+public class DeviceInstallationServiceTest {
+	
+	@Mock
+	private OrganisationDomainService organisationDomainService;
+	
+	@Mock
+	private DeviceDomainService deviceDomainService;
+	
+	@Mock
+	private OsgpCoreRequestMessageSender osgpCoreRequestMessageSender;
+	
+	@Mock
+	private WebServiceResponseMessageSender webServiceResponseMessageSender;
+	
+	@Mock
+	private DomainCoreMapper domainCoreMapper;
+	
+	@Mock
+	private SsldRepository ssldRepository;
+	
+	@InjectMocks
+	private DeviceInstallationService deviceInstallationService;
+
+	@Captor
+	private ArgumentCaptor<ResponseMessage> argumentResM;
+
+	@Captor
+	private ArgumentCaptor<RequestMessage> argumentReqM;
+
+	@Captor
+	private ArgumentCaptor<String> argumentStringOne;
+
+	@Captor
+	private ArgumentCaptor<String> argumentStringTwo;
+
+	@Captor
+	private ArgumentCaptor<Integer> argumentInt;
+	
+	@Test
+	public void testGetStatusTestDeviceTypeIsNotLmdType() throws FunctionalException {
+		final Device device = Mockito.mock(Device.class);
+		when(device.getIpAddress()).thenReturn("testIp");
+		when(this.deviceDomainService.searchActiveDevice("testDevice", ComponentType.DOMAIN_CORE)).thenReturn(device);
+		
+		this.deviceInstallationService.getStatus("testOrganisation", "testDevice", "testUid", "testMessageType", 1);
```

Comment:
Since we are going to check the values in the assert part of the tests, it is clearer to create constants for testIp, testUid, testOrganisation, testDevice and testMessageType. Furthmore, create a constant for testMessagePriority with value 1